pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -100,6 +101,65 @@ def test_rpm(self):
         )
         self.assertDictEqual(get_added_content_summary(repo), {})
 
+    def test_rpm_kickstart(self):
+        """Sync repositories with the rpm plugin.
```

Comment:
It would be nice to update docstring with the related issue.

```.rst

        This test targets the following issue:


        `Pulp #5201 <https://pulp.plan.io/issues/5201>`_
```